packages/neon/neon/lib/src/router.dart

```diff
@@ -1,41 +1,118 @@
 part of '../neon.dart';
```

Comment:
As this file will generate some code I'd say let's detach it from the main file.